.pylintrc

```diff
@@ -151,7 +151,9 @@ disable=raw-checker-failed,
         suppressed-message,
         useless-suppression,
         deprecated-pragma,
-        use-symbolic-message-instead
+        use-symbolic-message-instead,
+        invalid-name,
+        unnecessary-pass
```

Comment:
We don't need this - I think some pass statements are necessary, some aren't. pylint can tell. I've pushed a commit ([001fedc](https://github.com/CITCOM-project/CausalTestingFramework/pull/99/commits/001fedc3d448489b36a1f29c20506e7b9e3b463a)) so hopefully I'm not wrong.

Reply:
Ahh okay, I see that it is unnecessary when there are docstrings present as it's not 'empty code'. Personally I like them as they imply that the method is incomplete / needs filling in. I'm happy either way and to go with what the linter wants.